.github/workflows/mac_release.yml

```diff
@@ -0,0 +1,75 @@
+name: Mac Releases
```

Comment:
This is the actual workflow that builds Mac release packages for us now. Key points:

- Runs on Mac 10.15 (but we target 10.13 SDK)
- Is run only on tags, not regular commits
- Builds EnergyPlus against Python 3.7
- Builds and packages up the expected installer types: IFW, and TAR